### DISCUSSION TITLE
Shutdown Issue with latest Deluge Installation

### DISCUSSION THREAD

**Discussion by skl283** [tteck/Proxmox#958] [Help Needed]
I've installed deluge with the latest lxc-v5 Script and if i shutdown the lxc, it hangs several minutes (i think till a timeout is reached) - the system is only installed via script, nothing changed. I only get a blank/black screen - nothing happens.

Don't know how to find the issue - every other Script or LXC from you works (without the shutdown issue).

Do you have an idea what this could be the problem or to get more information to solve this?

PVE Versions are all updatet and Host is fresh reboot ;-)

**Comment by tteck** [comment]
I noticed that also. It seems it's been an ongoing issue, https://github.com/linuxserver/docker-deluge/issues/105
in our case, the `systemctl stop deluged.service` is the issue stopping. I'll see if I can come up with a remedy.
Of course you can use "Stop" instead of "Shutdown"

**Reply by skl283** [reply]
in deluged.service the timeout is set to 300s - changing/lowering this values helps to solve this symptom (not the issue).

`[Service]
Type=simple
UMask=007
ExecStart=/usr/local/bin/deluged -d
Restart=on-failure
TimeoutStopSec=300`

https://bbs.archlinux.org/viewtopic.php?pid=1892342 explains the issue.

**Comment by skl283** [comment]
If it's in the deluge code, you could perhaps change deluge to qbittorrent? Thats not a solution for hardcore deluge enthusiasts but could work a bit flawlessier?!

**Comment by skl283** [comment]
in my ovbservation/in my case zigbee2mqtt has also issues to shutdown clean/immediately.

**Reply by tteck** [reply]
No issues with Zigbee2MQTT.

**Reply by AlexGeek77** [reply]
I've had issues with the update process but it hasn't been repeatable. Usually the service fails to restart and I have to restart it via systemctl or reboot the container. Currently I have a weird issue where it doesn't show the webpage on Firefox but is fine on Safari. That's likely some other unrelated error.

**Comment by kvmgz** [comment]
Is there an update on this? 
I still have this issue.
